src/lib/containers/CopyToClipboardInput.tsx

```diff
@@ -66,7 +66,7 @@ export const CopyToClipboardInput: React.FunctionComponent<CopyToClipboardInputP
         ></FormControl>
         <Button
           variant="default"
-          className="SRC-copyToClipboardIcon"
+          id="SRC-copyToClipboardButton"
```

Comment:
Ok to have multiple elements with the same id on the page (I thought this was discouraged)?  If this is to increase css specificity score, there may be other ways (include element and css class name?).